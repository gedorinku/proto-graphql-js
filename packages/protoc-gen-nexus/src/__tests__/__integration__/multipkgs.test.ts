import { testSchemaGeneration } from "../__helpers__/process.test.helper";

testSchemaGeneration(["multipkgs/subpkg1", "multipkgs/subpkg2"], "protobufjs");
testSchemaGeneration(
  ["multipkgs/subpkg1", "multipkgs/subpkg2"],
  "native protobuf"
);
