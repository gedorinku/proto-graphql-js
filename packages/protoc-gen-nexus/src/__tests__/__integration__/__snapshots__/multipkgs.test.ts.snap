// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`schema generation with native protobuf can make GraphQL schema from generated DSLs: schema.graphql 1`] = `
"### This file was generated by Nexus Schema
### Do not make changes to this file directly


\\"\\"\\"\\"\\"\\"
type MessageWithSubpkg {
  \\"\\"\\"\\"\\"\\"
  enum: SubpkgEnum

  \\"\\"\\"\\"\\"\\"
  message: SubpkgMessage
}

\\"\\"\\"\\"\\"\\"
input MessageWithSubpkgInput {
  \\"\\"\\"\\"\\"\\"
  enum: SubpkgEnum

  \\"\\"\\"\\"\\"\\"
  message: SubpkgMessageInput
}

type Query {
  ok: Boolean!
}

\\"\\"\\"\\"\\"\\"
enum SubpkgEnum {
  BAR
  FOO
}

\\"\\"\\"\\"\\"\\"
type SubpkgMessage {
  \\"\\"\\"\\"\\"\\"
  body: String!
}

\\"\\"\\"\\"\\"\\"
input SubpkgMessageInput {
  \\"\\"\\"\\"\\"\\"
  body: String!
}
"
`;

exports[`schema generation with protobufjs can make GraphQL schema from generated DSLs: schema.graphql 1`] = `
"### This file was generated by Nexus Schema
### Do not make changes to this file directly


\\"\\"\\"\\"\\"\\"
type MessageWithSubpkg {
  \\"\\"\\"\\"\\"\\"
  enum: SubpkgEnum

  \\"\\"\\"\\"\\"\\"
  message: SubpkgMessage
}

\\"\\"\\"\\"\\"\\"
input MessageWithSubpkgInput {
  \\"\\"\\"\\"\\"\\"
  enum: SubpkgEnum

  \\"\\"\\"\\"\\"\\"
  message: SubpkgMessageInput
}

type Query {
  ok: Boolean!
}

\\"\\"\\"\\"\\"\\"
enum SubpkgEnum {
  BAR
  FOO
}

\\"\\"\\"\\"\\"\\"
type SubpkgMessage {
  \\"\\"\\"\\"\\"\\"
  body: String!
}

\\"\\"\\"\\"\\"\\"
input SubpkgMessageInput {
  \\"\\"\\"\\"\\"\\"
  body: String!
}
"
`;
